### DISCUSSION TITLE
What is the purpose of the resources folder?

### DISCUSSION THREAD

**Discussion by wpd** [pyscaffold/pyscaffold#745] [Best Practices & Workflows]
...and what are the best practices for using it?

I thought I could put a bitmap file in there and access it using PIL. but I can't figure out how to access the file.
It seems like I should be using `importlib.resources`, but, since the `resources` folder is outside of the `src` folder, I can't figure out how to make that work.

I tend to run quick tests of my PyScaffold generated application with something like:
```
$ python src/project/skeleton.py
```
(or, technically, VSCode does that on my behalf when I press F5).  Should I be doing something different so that Python can find the file I put in `resources`?

Any thoughts or recommendations?

--wpd

**Comment by wpd** [comment]
Whoops... never mind... I just realized that I created the `resources` folder, not PyScaffold! 🤦

**Reply by abravalheri** [reply]
Ok.

For using `importlib.resources`, you should place resource files inside your package folder, togheter with your `.py` files.

**Reply by wpd** [reply]
Thank you.  I couldn't figure out how to make that work with the `resources` folder was parallel to my package (aka `src`) folder.  I thought PyScaffold had placed that empty folder there for me... but I finally realized that _I_ put that empty folder there for me and that's why I couldn't figure out how to make things work.

Load gun... shoot foot... apologize for the loud noise.

--wpd
